### ISSUE TITLE
HDMI capture device audio

### ISSUE THREAD

**Issue by patmu** [windows10luke/V4L2-to-NDI#9]
Hi

Is audio processed using a HDMI capture device as well?

Best regards, Patrick

**Comment by lplassman** [comment]
No, this is video only for now. I haven't had time to add in an audio component.

**Comment by patmu** [comment]
Thanks for the answer. Hopefully you'll find time to add this feature sooner or later.
